sechub-solution/helm/sechub-server/templates/deployment.yaml

```diff
@@ -149,7 +149,7 @@ spec:
         - name: SECHUB_SERVER_SSL_KEYSTORE_TYPE
           value: "PKCS12"
         - name: SECHUB_SERVER_SSL_KEYSTORE_LOCATION
-          value: /home/javarun/{{ .Values.server.ssl.keystoreAlias }}.p12
+          value: /sechub/{{ .Values.server.ssl.keystoreAlias }}.p12
```

Comment:
If we want to use certificates in the future, we probably will mount them. For mounting it would be nice to have a directory like `/sechub/certs/xyz.p12` or `/certs/xyz.p12`. The mount point should be clear and it should be difficult to overwrite any existing directory.

Reply:
SSL certificates are already working.
`/sechub/{{ .Values.server.ssl.keystoreAlias }}.p12` is a symlink which is mounted in line 93f.
See also `run.sh` lines 112ff.

Why a symlink?
Because if the file `$SECHUB_SERVER_SSL_KEYSTORE_LOCATION` is absent, then Spring will create a self signed certificate. This is good if no certificate is provided.